shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -409,5 +419,23 @@ TEST(FlutterWindowsEngine, GetExecutableName) {
   EXPECT_EQ(engine->GetExecutableName(), "flutter_windows_unittests.exe");
 }
 
+// Ensure that after setting or resetting the high contrast feature,
+// the corresponding status flag can be retrieved from the engine.
+TEST(FlutterWindowsEngine, UpdateHighContrastFeature) {
```

Comment:
Could we verify that the embedder API was called with the expected values?

Reply:
+1. Check out this snippet for how to inject a custom handler for the embedder API call:
https://github.com/flutter/engine/blob/ebf65036d1bf0d7fefc55f6d1e21f61f85455752/shell/platform/windows/flutter_windows_engine_unittests.cc#L80-L92